src/Ext/TechnoType/Body.cpp

```diff
@@ -153,6 +153,9 @@ void TechnoTypeExt::ExtData::LoadFromINIFile(CCINIClass* const pINI)
 
 	this->ShieldType.Read(exINI, pSection, "ShieldType", true);
 
+	this->OnAmmoDepletion_AutoDeploy.Read(exINI, pSection, "OnAmmoDepletion_AutoDeploy");
+	this->OnAmmoDepletion_DeployUnlockAmount.Read(exINI, pSection, "OnAmmoDepletion_DeployUnlockAmount");
```

Comment:
```suggestion
	this->OnAmmoDepletion_AutoDeploy.Read(exINI, pSection, "OnAmmoDepletion.AutoDeploy");
	this->OnAmmoDepletion_DeployUnlockAmount.Read(exINI, pSection, "OnAmmoDepletion.DeployUnlockAmount");
```

Replace underscores with dots for INI entries.